docs/_partials/reverification-route-handler.mdx

```diff
@@ -0,0 +1,20 @@
+The following example uses the [`has()`](/docs/references/nextjs/auth#has) helper to check if the user has verified their credentials within a specific time period. The `strict` configuration sets the time period to 10 minutes. If the user hasn't verified their credentials within 10 minutes, the `reverificationError` utility is used to return an error and initiate the session reverification flow on the client-side.
+
+```tsx {{ filename: 'app/api/reverification-example/route.ts' }}
+import { auth, reverificationErrorResponse } from '@clerk/nextjs/server'
+
+export const POST = async (req: Request) => {
+  const { has } = await auth()
+
+  // Check if the user has *not* verified their credentials within the past 10 minutes.
+  const shouldUserRevalidate = !has({ reverification: 'strict' })
+
+  // If the user hasn't reverified, return an error with the matching configuration (e.g., `strict`)
+  if (shouldUserRevalidate) {
+    return reverificationError('strict')
```

Comment:
Inside route handlers devs should use `reverificationErrorResponse` instead of `reverificationError`.

```suggestion
    return reverificationErrorResponse('strict')
```